source/BoxCastAPI.brs

```diff
@@ -108,10 +108,12 @@ function ContentNodeFromBroadcastJson(broadcast)
     starts_at: ValidStr(broadcast.starts_at)
   })
 
-  if broadcast.transcoder_profile = "720p"
+  broadcast720 = CreateObject("roRegex", "^720p", "")
```

Comment:
Could alternatively check  `if Left(broadcast.transcoder_profile, 4) = "720p"` but this looks like it will work!

Reply:
I was nervous about what that would do in the 1080p case if the string is shorter than the requested digits.

Reply:
Yeah, this is fine.  I don't know how `Left` behaves either in that case and the docs are scant.